tensorflow/compiler/xla/service/gpu/tests/gpu_input_fusible_slice_test.cc

```diff
@@ -63,12 +63,17 @@ TEST_F(GpuSliceInputFusionTest, InputFusionWithOnlyOneSlice) {
   auto hlo_module =
       ParseAndReturnVerifiedModule(kHloString, ConfigWithoutLayoutAssignment())
           .ValueOrDie();
-  CompileAndVerifyIr(std::move(hlo_module),
-                     R"(
+  auto expected_ir = is_built_with_rocm_ ? R"(
+; CHECK-LABEL: define amdgpu_kernel void @fusion
```

Comment:
Could we just optionally match amdgpu_kernel using a regexp instead of duplication?

Reply:
Same applies below.

Reply:
this may not be viable.

We want to check for presence of `amdgpu_kernel` in ROCm mode, so having a regexp that optionally matches `amdgpu_kernel` will not work, since it will pass even if `amdgpu_kernel` is not present!

can we leave this as is?

Reply:
OK sure, in general we have to weigh in the risk of one of the branches getting stale (recall that currently we do not run ROCM tests internally, so it's possible that someone would update the nvidia branch, but not the ROCM branch) vs. the exact pattern you want to match. So here I would say that matching `amdgpu_kernel` is not important enough to have two branches, but since the pattern is small I don't care that much either.

Reply:
@whchung , what say you?

If you are okay with it, I can push out new PR which makes the check for `amdgpu_kernel` optional?